src/view/border.rs

```diff
@@ -243,32 +243,51 @@ mod tests {
 
     struct AppState;
 
-    #[test]
-    fn simple_border_test() {
-        let sut = Arc::new("some text".fg(Color::Cyan).border(()));
-        let buffer = render_view(
-            Size {
-                width: 15,
-                height: 5,
-            },
-            sut,
-            AppState,
-        );
-
-        insta::assert_debug_snapshot!(buffer);
+    #[tokio::test]
+    async fn simple_border_test() {
+        let _guard = crate::test_helper::init_tracing("simple_border_test").unwrap();
```

Comment:
Not sure whether we need to init tracing for tests at all. But it may be helpful for debugging of course.

Reply:
I actually found it quite helpful for debugging tests.
The main reason I added it was to ensure it works, because of my previous PR, where I had to change App::run() because the way tracing was initialized was incompatible with test runs.
Currently it's mostly useful as an example how to initialize tracing in case someone wants to use it from tests.

Reply:
Yeah I have no strong feelings about it. Could be helpful as you say.